en/mozorg/about/governance/policies/participation.ftl

```diff
@@ -0,0 +1,107 @@
+participation-reporting = Reporting
+participation-if-you-believe-youre = If you believe you’re experiencing unacceptable behavior that will not be tolerated as outlined above, <a href="{ $hotline }">please use our hotline to report</a>. Reports go directly to { -brand-name-mozilla }’s Employment Counsel and HR People Partners and are triaged by the Community Participation Guidelines Response Lead.
+participation-after-receiving-a = After receiving a concise description of your situation, they will review and determine next steps. In addition to conducting any investigation, they can provide a range of resources, from a private consultation to other community resources. They will involve other colleagues or outside specialists (such as legal counsel), as needed to appropriately address each situation.
+participation-additional-resources = Additional Resources: <a href="{ $url }">How to Report</a>
+participation-questions-cpg-questionsmozillacom = Questions: <a href="{ $url }">cpg-questions@mozilla.com</a>
+participation-please-also-report = Please also report to us if you observe a potentially dangerous situation, someone in distress, or violations of these guidelines, even if the situation is not happening to you.
+participation-if-you-feel-you-have = If you feel you have been unfairly accused of violating these guidelines, please follow the same reporting process.
+participation-mozilla-spaces = { -brand-name-mozilla } Spaces
+participation-each-physical-or-virtual = Each physical or virtual { -brand-name-mozilla } space shall have a designated contact.
+participation-mozilla-events = { -brand-name-mozilla } Events
+participation-all-mozilla-events = All { -brand-name-mozilla } events will have designated a specific safety guideline with emergency and anti-abuse contacts at the event as well as online. These contacts will be posted prominently throughout the event, and in print and online materials. Event leaders are requested to speak at the event about the guidelines and to ask participants to review and agree to them when they sign up for the event.
+participation-reports-will-receive = Reports will receive an email notice of receipt. Once an incident has been investigated and a decision has been communicated to the relevant parties, all have the opportunity to appeal this decision by sending an email to <a href="{ $mailto_questions }">cpg-questions@mozilla.com</a>.
+participation-ask-questions = Ask questions
+participation-everyone-is-encouraged = Everyone is encouraged to ask questions about these guidelines. If you are organizing an event or activity, reach out for tips building inclusion for your event, activity or space. Your input is welcome and you will always get a response within 24 hours (or on the next weekday, if it is the weekend) if you reach out to <a href="{ $mailto_questions }">cpg-questions@mozilla.com</a>. Please <a href="{ $changelog }">review this change log</a> for updates to this document.
```

Comment:
A word is missing after `tips`: `... reach out for tips building inclusion for your event ...`